Dockerfile

```diff
@@ -13,23 +13,23 @@ WORKDIR /app
 # Copy the package.json file from the local directory to /app in the container
 COPY ./app/package.json ./
 
-# Copy the yarn.lock file from the local directory to /app in the container
-COPY yarn.lock ./
+# Copy the pnpm-lock.yaml file from the local directory to /app in the container
+COPY pnpm-lock.yaml ./
```

Comment:
🤔 @crherman7 do we still need copy lock file?

Reply:
yeah lock file is important in-case someone downstream releases a breaking change that doesn't abide by semver versioning